elasticache.tf

```diff
@@ -14,6 +14,6 @@ resource "aws_elasticache_cluster" "redis_version_non_compliant" {
   node_type            = "cache.m4.large"
   num_cache_nodes      = 1
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure that ElastiCache for Redis version is compliant with AWS PCI DSS requirements**
 
**Vulnerability**
ElastiCache for Redis version is not compliant with AWS PCI DSS requirements
 
**Remediation**
To create a PCI-Compliant ElastiCache for Redis cluster, you must use the latest Redis engine version 4.0.10 or higher and current generation node types